src/main/java/org/wso2/carbon/connector/azure/storage/BlobEraser.java

```diff
@@ -40,19 +40,23 @@
 public class BlobEraser extends AbstractConnector {
 
     public void connect(MessageContext messageContext) {
-        if (messageContext.getProperty(AzureConstants.ACCOUNT_NAME) == null || messageContext.getProperty(
-                AzureConstants.ACCOUNT_KEY) == null || messageContext.getProperty(AzureConstants.CONTAINER_NAME) == null
-                || messageContext.getProperty(AzureConstants.FILE_NAME) == null){
+        if (messageContext.getProperty(AzureConstants.PROTOCOL) == null ||
```

Comment:
This makes the protocol param mandatory. We need to make it optional and default to http. Could you please modify the PR as such?